backward.hpp

```diff
@@ -3797,11 +3797,17 @@ class SourceFile {
   }
 #endif
 
+  // Allow adding to paths gotten from BACKWARD_CXX_SOURCE_PREFIXES after loading the
+  // library; this can be useful when the library is loaded when the locations are unknown
+  static void add_paths_to_env_variable_impl(const std::string & to_add) {
+    get_mutable_paths_from_env_variable().push_back(to_add);
+  }
+
 private:
   details::handle<std::ifstream *, details::default_delete<std::ifstream *>>
       _file;
 
-  std::vector<std::string> get_paths_from_env_variable_impl() {
+  static std::vector<std::string> get_paths_from_env_variable_impl() {
```

Comment:
`static` makes no sense here. What were you trying to do?

Reply:
This was to enable it being called from a static `get_mutable_paths_from_env_variable`